content/en/docs/ops/integrations/spire/index.md

```diff
@@ -0,0 +1,385 @@
+to check issued identities.
+
+### Option 2: Manual Registration
+
+SPIRE Server is able to verify a group of values to improve workload attestation security
+robustness.
+
+1. To generate an entry for an ingress-gateway with a set of selectors such as the
+   pod name and pod-uid:
+
+
+{{< text bash >}}
+$ INGRESS_POD=$(kubectl get pod -l istio=ingressgateway -n istio-system -o jsonpath="{.items[0].metadata.name}")
+$ INGRESS_POD_UID=$(kubectl get pods -n istio-system $INGRESS_POD -o jsonpath='{.metadata.uid}')
+{{< /text >}}
+
+2. Get the spire-server pod:
+
+{{< text bash >}}
+$ SPIRE_SERVER_POD=$(kubectl get pod -l app=spire-server -n spire -o jsonpath="{.items[0].metadata.name}")
+{{< /text >}}
+
+3. Attest the SPIRE Agent running on the node
+   {{< text bash >}}
+   $ kubectl exec -n spire $SPIRE_SERVER_POD -- \
+   /opt/spire/bin/spire-server entry create \
+   -spiffeID spiffe://example.org/ns/spire/sa/spire-agent \
+   -selector k8s_psat:cluster:demo-cluster \
+   -selector k8s_psat:agent_ns:spire \
+   -selector k8s_psat:agent_sa:spire-agent \
+   -node -socketPath /run/spire/sockets/server.sock
+   
+   Entry ID         : d38c88d0-7d7a-4957-933c-361a0a3b039c
+   SPIFFE ID        : spiffe://example.org/ns/spire/sa/spire-agent
+   Parent ID        : spiffe://example.org/spire/server
+   Revision         : 0
+   TTL              : default
+   Selector         : k8s_psat:agent_ns:spire
+   Selector         : k8s_psat:agent_sa:spire-agent
+   Selector         : k8s_psat:cluster:demo-cluster
+   {{</ text >}}
+
+4. and then register an entry for istio-ingressgateway pod.
+   {{< text bash >}}
+   $ kubectl exec -n spire $SPIRE_SERVER_POD -- \
+   /opt/spire/bin/spire-server entry create \
+   -spiffeID spiffe://example.org/ns/istio-system/sa/istio-ingressgateway-service-account \
+   -parentID spiffe://example.org/ns/spire/sa/spire-agent \
+   -selector k8s:sa:istio-ingressgateway-service-account \
+   -selector k8s:ns:istio-system \
+   -selector k8s:pod-uid:$INGRESS_POD_UID \
+   -dns $INGRESS_POD \
+   -dns istio-ingressgateway.istio-system.svc \
+   -socketPath /run/spire/sockets/server.sock
+
+   Entry ID         : 6f2fe370-5261-4361-ac36-10aae8d91ff7
+   SPIFFE ID        : spiffe://example.org/ns/istio-system/sa/istio-ingressgateway-service-account
+   Parent ID        : spiffe://example.org/ns/spire/sa/spire-agent
+   Revision         : 0
+   TTL              : default
+   Selector         : k8s:ns:istio-system
+   Selector         : k8s:pod-uid:63c2bbf5-a8b1-4b1f-ad64-f62ad2a69807
+   Selector         : k8s:sa:istio-ingressgateway-service-account
+   DNS name         : istio-ingressgateway.istio-system.svc
+   DNS name         : istio-ingressgateway-c48554dd6-cff5z
+   {{</ text >}}
+
+5. Deploy an example workload:
+
+{{< text bash >}}
+$ istioctl kube-inject --filename @samples/httpbin/httpbin.yaml | kubectl apply -f -

```

Comment:
Does this work without adding the annotation for the sidecar injection?

Reply:
It doesn't work, why do you ask?

Reply:
Assuming the user perspective here, I'd probably expect this deployment to be working as if it were correctly integrated with SPIRE and Istio (we're showing that SPIRE acknowledged this service, but Istio will not be using SPIRE issued identities when doing mTLS). 
I think at least being clear about this, or mentioning about the need for annotations to inject the custom sidecar template (probably somewhere else) would be good enough. IMO this information is pretty hidden for a basic deployment requirement (can be found in the `istio-config.yaml` provided in the other PR).

Reply:
I see what you mean, assuming that the user installed Istio using the patch from Install Istio section, every `istioctl` injection will apply the `spire` template to the deployment spec of the workload. I will add an aditional reminder that the workload will need to access the csi volume and link to the spire template of the `istio-config`, or tell them to add the csi volume to the their workload spec.